src/data/real/ereal.lean

```diff
@@ -251,6 +284,21 @@ def ne_top_bot_equiv_real : ({⊥, ⊤} : set ereal).compl ≃ ℝ :=
   end,
   right_inv := λ x, by simp }
 
+
+/-! ### Supremum and Infimum -/
+
+lemma ereal.le_Sup {α : set ereal} {x : ereal} (h : x ∈ α) : x ≤ Sup α :=
+  @_root_.le_Sup ereal _ _ _ h
+
+lemma ereal.Inf_le  {α : set ereal} {x : ereal} (h : x ∈ α) : Inf α ≤ x :=
+  @_root_.Inf_le ereal _ _ _ h
```

Comment:
What's the idea behind having these lemmas vs just using `_root_.le_Sup` directly?

Reply:
When I tried, ```library_search``` couldn't find the answer for me. Maybe that's not a good enough reason?